src/App.jsx

```diff
@@ -1,131 +1,49 @@
 import { Layout, Space } from 'antd'
 const { Header, Footer, Sider, Content } = Layout
-
-const headerStyle = {
-  fontFamily: 'Georgia',
-  textAlign: 'center',
-  fontSize: '30px',
-  color: '#fff',
-  height: 120,
-  lineHeight: '120px',
-  paddingInline: 50,
-  backgroundColor: '#af2127',
-}
-
-const contentStyle = {
-  fontFamily: 'sans-serif',
-  fontSize: 16,
-  minHeight: 120,
-  lineHeight: '35px',
-  color: '#260000',
-  backgroundColor: '#fcf5f5',
-  paddingBottom: 5,
-}
-
-const siderStyle = {
-  textAlign: 'center',
-  lineHeight: '90px',
-  color: '#000',
-  backgroundColor: '#fff0f0',
-  paddingTop: 20,
-}
-
-const footerStyle = {
-  textAlign: 'center',
-  color: '#fff',
-  backgroundColor: '#af2127',
-}
-
-const pStyle = {
-  marginRight: 40,
-  marginLeft: 40,
-  marginTop: 25,
-  marginBottom: 25,
-}
-
-const h2Style = {
-  textAlign: 'center',
-  marginTop: 25,
-}
-
-const buttonStyle = {
-  fontFamily: 'sans-serif',
-  textAlign: 'center',
-  padding:10,
-  paddingRight:20,
-  paddingLeft:20,
-  margin: 20,
-  border: 0,
-  borderRadius: '25px',
-  color: '#fff',
-  backgroundColor: '#af2127',
-}
-
-const imgStyle = {
-  width: '100%',
-  padding: 0,
-  margin:0,
-}
-
-const citiesStyle = {
-  display: 'flex',
-  flexDirection: 'row',
-  justifyContent: 'space-around',
-  height: '60px',
-  paddingRight: 225,
-  paddingLeft: 225,
-  alignItems: 'center',
-  fontSize: 16,
-}
+import './App.css'
 
 function App() {
   return (
   <Space
     direction="vertical"
-    style={{
-      width: '100%',
-      margin: 0
-
-    }}
+    className='space'
     size={[0, 48]}
   >
     <Layout>
-      <Sider style={siderStyle}>
-        <button style={buttonStyle}>Главная страница</button>
-        <button style={buttonStyle}>Рестораны</button>
-        <button style={buttonStyle}>Заказать доставку</button>
-        <button style={buttonStyle}>Вакансии</button>
-        <button style={buttonStyle}>Оплата</button>
-        <button style={buttonStyle}>Контакты</button>
+      <Sider className='sider'>
+        <button>Главная страница</button>
+        <button>Рестораны</button>
+        <button>Заказать доставку</button>
+        <button>Вакансии</button>
+        <button>Оплата</button>
+        <button>Контакты</button>
```

Comment:
Please use `Menu` component from `antd`
```suggestion
          <Menu
            items={[
              { key: "1", label: "Label 1" },
              { key: "2", label: "Label 2" },
              { key: "3", label: "Label 3" },
              { key: "4", label: "Label 4" },
              { key: "5", label: "Label 5" },
              { key: "6", label: "Label 6" },
            ]}
          />
```

Reply:
Fixed